blocks/tabs/tabs.js

```diff
@@ -1,62 +1,81 @@
+  }
 
-  // decorate tabs and tabpanels
-  const tabs = [...block.children].map((child) => child.firstElementChild);
-  tabs.forEach((tab, i) => {
-    if (i === 0) {
-      if (tab.textContent) {
-        block.classList.add(tab.textContent);
-        tab.remove();
+  const id = toClassName(tab.textContent);
+  const tabpanel = block.children[index];
+
+  if (tabpanel.querySelector('a')) {
+    const link = tabpanel.querySelector('a').href;
+
+    if (link.includes('/fragments/')) {
+      const url = new URL(link);
+      const fragmentPath = url.pathname;
+
+      const fragmentBlock = await loadFragment(fragmentPath);
+      if (fragmentBlock) {
+        const lastChild = tabpanel.lastElementChild;
+        const fragmentChild = fragmentBlock.querySelector('.section');
+        if (fragmentChild) {
+          lastChild.innerHTML = fragmentChild.innerHTML;
+        }
       }
-      return;
-    }
-    const id = toClassName(tab.textContent);
-
-    // decorate tabpanel
-    const tabpanel = block.children[i];
-    tabpanel.className = 'tabs-panel';
-    tabpanel.id = `tabpanel-${id}`;
-    tabpanel.setAttribute('aria-hidden', !!i);
-    tabpanel.setAttribute('aria-labelledby', `tab-${id}`);
-    tabpanel.setAttribute('role', 'tabpanel');
-    if (!hasWrapper(tabpanel.lastElementChild)) {
-      tabpanel.lastElementChild.innerHTML = `<p>${tabpanel.lastElementChild.innerHTML}</p>`;
     }
+  }
+  tabpanel.className = 'tabs-panel';
+  tabpanel.id = `tabpanel-${id}`;
+  tabpanel.setAttribute('aria-labelledby', `tab-${id}`);
+  tabpanel.setAttribute('role', 'tabpanel');
+  tabpanel.setAttribute('aria-hidden', index !== 1);
+  if (!hasWrapper(tabpanel.lastElementChild)) {
+    tabpanel.lastElementChild.innerHTML = `<p>${tabpanel.lastElementChild.innerHTML}</p>`;
+  }
 
-    // build tab button
-    const button = document.createElement('button');
-    button.className = 'tabs-tab';
-    button.id = `tab-${id}`;
-    button.innerHTML = tab.innerText;
-    button.setAttribute('aria-controls', `tabpanel-${id}`);
-    button.setAttribute('aria-selected', !i);
-    button.setAttribute('role', 'tab');
-    button.setAttribute('type', 'button');
-
-    //  Click events
-    button.addEventListener('click', () => {
-      block.querySelectorAll('[role=tabpanel]').forEach((panel) => {
-        panel.setAttribute('aria-hidden', true);
-      });
-      tablist.querySelectorAll('button').forEach((btn) => {
-        btn.setAttribute('aria-selected', false);
-      });
-      tabpanel.setAttribute('aria-hidden', false);
-      button.setAttribute('aria-selected', true);
+  const button = document.createElement('button');
+  button.className = 'tabs-tab';
+  button.id = `tab-${id}`;
+  button.innerHTML = tab.innerText;
+  button.setAttribute('aria-controls', `tabpanel-${id}`);
+  button.setAttribute('aria-selected', index === 1);
+  button.setAttribute('role', 'tab');
+  button.setAttribute('type', 'button');
+
+  button.addEventListener('click', () => {
```

Comment:
@Nandhiniraghu can we add focus on selected tab for the accessibility `tabpanel.focus();` ?

Reply:
By default, focus will applied to the button when clicking. If not then we have to set the focus.

Reply:
As Button Component is used, by default Focus is applied